frontend/src/components/Sidebar/index.tsx

```diff
@@ -35,32 +35,32 @@ export const Sidebar = () => {
    */
   const adminLinks = [
     {
-      name: "Contracts",
-      link: "admin/contracts",
+      name: "Contacts",
+      link: "/admin/contacts",
     },
     {
       name: "Suppliers",
-      link: "admin/suppliers",
+      link: "/admin/suppliers",
     },
     {
       name: "Subcontractors",
-      link: "admin/subcontractors",
+      link: "/admin/subcontractors",
     },
     {
       name: "Resources",
-      link: "admin/resources",
+      link: "/admin/resources",
     },
     {
       name: "Ministries / Org Name",
-      link: "admin/ministries",
+      link: "/admin/ministries",
     },
     {
       name: "User Management",
-      link: "admin/userManagement",
+      link: "/admin/userManagement",
```

Comment:
was there a reason you used userManagement as opposed to users

Reply:
Thanks for catching this mistake!! My mind must have been thinking about the "name" at the time.  I will fix this.